OpenIPSL/Electrical/Solar/KTH/Auxiliary/Capacitor.mo

```diff
@@ -15,9 +15,9 @@ model Capacitor
         origin={-120.0,-0.0},
         extent={{-20.0,-20.0},{20.0,20.0}})));
   Real x(start=Udc0/(1000*UdcN));
```

Comment:
@marcelofcastro  Could it be that this factor 1000 also needs to get removed?

Reply:
good catch